pkgs/fc/ceph/src/fc/ceph/tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import fc.ceph.maintenance as maintSub
+import pytest
+
+
+@pytest.fixture(params=[maintSub.jewel, maintSub.luminous, maintSub.nautilus])
+def maintenance_manager_legacy(request):
```

Comment:
Uuuuuuhm. Is this actually used anywhere?

Reply:
No. https://github.com/flyingcircusio/fc-nixos/pull/679#discussion_r1206608070